src/app/component/shared/error-handler/error-handler.component.spec.ts

```diff
@@ -1,28 +1,37 @@
-import { Component, OnInit } from "@angular/core";
+import { ChangeDetectorRef, Component, OnInit } from "@angular/core";
 import {
   ComponentFixture,
   fakeAsync,
   TestBed,
   tick
 } from "@angular/core/testing";
 import { testBawServices } from "src/app/app.helper";
+import { APIErrorDetails } from "src/app/services/baw-api/api.interceptor";
 import { BawApiService } from "src/app/services/baw-api/base-api.service";
 import { SharedModule } from "../shared.module";
 import { ErrorHandlerComponent } from "./error-handler.component";
 
 @Component({
-  template: "<app-error-handler [errorCode]='errorCode'></app-error-handler>"
+  template: "<app-error-handler [error]='error'></app-error-handler>"
 })
 class TestErrorHandlerComponent implements OnInit {
-  errorCode: number;
+  error: APIErrorDetails;
 
-  constructor() {}
+  constructor(private ref: ChangeDetectorRef) {}
 
   ngOnInit() {
-    this.errorCode = 401;
+    this.error = {
+      status: 401,
+      message: "You need to log in or register before continuing."
+    };
```

Comment:
Should convert to `ApiErrorDetails` so that any future changes to the datatype will be detected.